dist/znk.infra.js

```diff
@@ -2771,9 +2813,11 @@
 
                 var StatsSrv = {};
 
-                function _getCategoryLookup() {
-                    return $injector.invoke(getCategoryLookup);
-                }
+                var _getCategoryLookup = function() {
+                    return $injector.invoke(getCategoryLookup).then(function(categoryMap){
+                        return categoryMap;
```

Comment:
the return wasn't there before, the return was straight from the injector invoke.